### DISCUSSION TITLE
Bitdevs en Español #004

### DISCUSSION THREAD

**Discussion by lucasdcf** [bitdevsespanol/bitdevsespanol.org#4] [General]
Hola chicos, creé esta discusión aquí para que envíen los temas que desean discutir en el próximo Bitdevs (en el dia 27/07 a las 19:00 UTC)
Si hay algun tema del meetup anterior que le gustaría discutir más porque tuvimos poco tiempo, avisen aquí también.
Envía también los enlaces. Y si hay algun tema que el que te sientes muy cómodo, puedes ser voluntario para presentar también :)
Si te gusta el tema del colega, puene un 👍

**Comment by a-mpch** [comment]
Sería interesante hablar de cómo alguien llega a ser mantenedor de Bitcoin-core y como es el proceso de dejar de serlo a raíz de Pieter deja de serlo y @glozow ahora lo es

**Comment by azorcode** [comment]
Hablar un poco mas de Taro y LNURL

**Comment by kleysc** [comment]
Lightning Pool en general, su usabilidad a través de Terminal Web o Flow de Voltage

**Comment by BlueeeMoon** [comment]
Me parece que un tema interesante sería hablar de las transacciones de **Bitcoin** parcialmente firmadas **PSBT.**

- Viene especificado en el [BIP-0174](https://github.com/bitcoin/bips/blob/master/bip-0174.mediawiki) 
- Me parece que podríamos hacer un ejercicio con esto utilzando `lnd. `
- Aquí un [artículo ](https://dinerosinreglas.com/psbt/) interesante.

Gracias.
